spec/classes/init_spec.rb

```diff
@@ -254,6 +254,20 @@
     end
   end
 
+  context 'with parameter firewallbackend' do
+    let(:params) do
+      {
+        firewallbackend: 'nftables'
+      }
+    end
+
+    it do
```

Comment:
could you please add similar under the context `'with defaults for all parameters'` that shows it `is_expected.not_to contain_augeas('firewalld::firewallbackend')`